src/components/molecules/Pagenation/index.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div v-if="totalPages">
+    <div>
+      <nav class="pagenation">
+        <ul class="pagenation__list">
+          <li
+            class="pagenation__list__btn pagenation__list__first"
+            :class="{ 'pagenation__list__disabled': 1 === currentPage }"
```

Comment:
[nits]

比較対象が左の方が見やすいです！
他の箇所も見返してみてください！

```suggestion
            :class="{ 'pagenation__list__disabled': currentPage === 1 }"
```

Reply:
f64cdf7d1b87eb7ad3e26739592c0bce46e6cf4e

読みやすいように currentPage === の形に修正しました